scripts/helpers.sh

```diff
@@ -62,3 +62,44 @@ command_exists() {
   local command="$1"
   command -v "$command" &> /dev/null
 }
+
+get_tmp_dir() {
+  echo "${TMPDIR:-/tmp}/tmux-$EUID-cpu"
```

Comment:
Not sure about non-Linux system (OSX, Cygwin).
( https://gist.github.com/rcmdnk/7851026 )

Reply:
I'd taken inspiration from the approach of `tmux-copycat` which does `${TMPDIR:-/tmp}/tmux-$EUID-copycat` but that gist looks more robust.

Reply:
```suggestion
  local tmpdir="${TMPDIR:-${TMP:-${TEMP:-/tmp}}}"
  if [ ! -d "$tmpdir" ]; then
    local tmpdir=~/tmp
    mkdir -p $tmpdir
  fi
  echo "$tmpdir/tmux-$EUID-cpu"
```